packages/tlschema/src/assets/TLImageAsset.ts

```diff
@@ -26,11 +26,11 @@ export const imageAssetValidator: T.Validator<TLImageAsset> = createAssetValidat
 	T.object({
 		w: T.number,
 		h: T.number,
-		fileSize: T.number,
 		name: T.string,
 		isAnimated: T.boolean,
 		mimeType: T.string.nullable(),
 		src: T.srcUrl.nullable(),
+		fileSize: T.number.optional(),
```

Comment:
whilst we're here, lets make this a `T.nonZeroInteger.optional()` maybe

Reply:
Will it always be an integer? Or nonZeroNumber ok?

Reply:
it's measured in bytes, and you can't have fractions of a byte, so i think so?

Reply:
on

ze